### PR TITLE
[TV][NUI] Fix build & warning errors of TV/NUI.

### PR DESCRIPTION
**Pull request by AchoWang**
test target:
PontusM TV
csapi-tizenfx-tv-10.0.0.17353+nui22134-1.noarch

Samples
**ChannelList** ： Pass
**FirstScreen** ： The sample can launch normally, but it seems cannot response to the remote control.
**MediaHubSample** ： Pass
**MovieLibrary** ： Pass
**VisualSample** ： Pass

### CODE REVIEW

File: TV/NUI/ChannelList/ChannelList/ChannelList.csproj [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
+﻿<Project Sdk="Tizen.NET.Sdk/1.1.8">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>tizen60</TargetFramework>
+    <TargetFramework>tizen10.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>CS0618</NoWarn>
```
Comment (dongsug-song) [inline TV/NUI/ChannelList/ChannelList/ChannelList.csproj:9]: it seems not good to blocking the warning. 
it is supposed to resolve the build error and warning basically.
please check again. thank you.

**Comment by dongsug-song** [comment]
@AchoWang
Hello,
could you update the description?
and could you share the test result whether this sample App runs properly in tizen_7.0 emulator?
thank you.

File: TV/NUI/ChannelList/ChannelList/ChannelList.csproj (hunk 1 above)
Comment (AchoWang) [reply TV/NUI/ChannelList/ChannelList/ChannelList.csproj:9]: > it seems not good to blocking the warning. it is supposed to resolve the build error and warning basically. please check again. thank you.

Hi, I checked the build process , and there have no warning info. Actually, 

public int Width
        {
            [Obsolete("Do not use this setter, that is deprecated in API8 and will be removed in API10. Use new Size2D(...) constructor")]
            set
            {
                Interop.Vector2.WidthSet(SwigCPtr, (float)value);
                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();

                callback?.Invoke(value, Height);
            }
            get
            {
                float ret = Interop.Vector2.WidthGet(SwigCPtr);
                if (NDalicPINVOKE.SWIGPendingException.Pending) throw new InvalidOperationException("FATAL: get Exception", NDalicPINVOKE.SWIGPendingException.Retrieve());
                return ClampToInt(ret);
            }
        }


when the setter is removed, the warning info will be disappeared.

**Review by dongsug-song — APPROVED** [review]
approved

**Pull request merged**